plugins/hls-floskell-plugin/src/Ide/Plugin/Floskell.hs

```diff
@@ -33,7 +33,7 @@ provider _ideState typ contents fp _ = liftIO $ do
     config <- findConfigOrDefault file
     let (range, selectedContents) = case typ of
           FormatText    -> (fullRange contents, contents)
-          FormatRange r -> (r, extractRange r contents)
+          FormatRange r -> (normalize r, extractRange r contents)
```

Comment:
It won't work without this change. I don't know why. All other formatters use `normalize` already.

Reply:
I assumed, floskell performed the normalisation internally. That, or no one is running the tests.